frontend/src/scenes/feature-flags/FeatureFlag.scss

```diff
@@ -61,4 +61,9 @@
         align-items: center;
         white-space: pre-wrap;
     }
+
+    .break {
```

Comment:
This should almost certainly be implemented as a Tailwind class (`basis-full h-0`) on the component

Reply:
will do thanks!